src/arch/x86/rte_atomic.h

```diff
@@ -217,9 +217,9 @@ static inline int rte_atomic32_dec_and_test(rte_atomic32_t *v)
 #endif
 
 #ifdef RTE_ARCH_I686
-#include "rte_atomic_32.h"
+#include "arch/x86/rte_atomic_32.h"
```

Comment:
where is this file rte_atomic_32.h? I don't see it in the PR.

Reply:
There was no such file in the original project. I thought this file was not needed. I find the file in the DPDK repo. https://github.com/DPDK/dpdk/blob/master/lib/librte_eal/common/include/arch/x86/rte_atomic_32.h. I can upload it if needed

Reply:
lets raise a separate PR for that.

Reply:
ok, I will raise it later